webpack.config.js

```diff
@@ -19,6 +19,7 @@ const EXPLORER_APP = {
   title: 'CoW Protocol Explorer',
   filename: 'index.html',
   envVars: {
+    ETH_NODE_URL: process.env.ETH_NODE_URL,
     INFURA_ID: process.env.INFURA_ID,
```

Comment:
Keep in mind that:
1. INFURA_ID takes precedence. If this is set, ETH_NODE_URL won't be used at all, AFAICT
2. If ETH_NODE_URL is chain specific, other chains might not work (I have not tested this but I assume that's what will happen)